From 793a9af03698d2d28c73fb5e0680237653379d05 Mon Sep 17 00:00:00 2001
From: "Philip, Avinash" <avinashphilip@ti.com>
Date: Sun, 11 Mar 2012 13:52:19 +0530
Subject: [PATCH 165/186] PWM: Remove unused data

Removes unused data in PWM platform data structure
	config_semaphore
	pwmss_module_usage_count
	config_mem_base

Signed-off-by: Philip, Avinash <avinashphilip@ti.com>
---
 include/linux/pwm/pwm.h |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/include/linux/pwm/pwm.h b/include/linux/pwm/pwm.h
index 3bab0a1..99c08f5 100644
--- a/include/linux/pwm/pwm.h
+++ b/include/linux/pwm/pwm.h
@@ -122,9 +122,6 @@ struct pwm_chan_attrib {
 
 struct pwmss_platform_data {
 	int channel_mask;
-	struct semaphore config_semaphore;
-	int pwmss_module_usage_count;
-	void *config_mem_base;
 	u8 version;
 	struct pwm_chan_attrib chan_attrib[PWM_CHANNEL];
 };
-- 
1.7.9.1

